### PR TITLE
Videos listing

### PR DESCRIPTION
**Pull request by omkarpatke**

### CODE REVIEW

File: src/Pages/HomePage/HomePage.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    gap: 1rem 1rem;
+    background-color: var(--bg-color);
+    padding: 1rem;
+    min-height: 100vh;
+}
+
+.video{
+   background-color: var(--white-color);
+   max-width: 300px;
+   max-height: 340px;
```
Comment (yaswanthmyneni) [inline src/Pages/HomePage/HomePage.css:32]: changes: give rem unit for value more than 15px

File: src/Pages/HomePage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <SideBar />
+    <div className="home-section">
+    <CategoryBar />
+    <div className='homePage-container'>
+    <div className='videos-container'>
+     {filteredVideos.map((video) => (
+       <div key={video._id} className='video'>
+         <img src={video.image} alt="videoImg" className='video-img' />
+         <div className='video-info-container'>
+         <div className="video-title">{video.title}</div>
```
Comment (yaswanthmyneni) [inline src/Pages/HomePage/HomePage.js:23]: suggestion: we can make a separate component of this

File: src/Pages/SignIn/SignIn.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+.login-card{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    padding: 1rem 3rem 2rem 3rem;
+    margin: 3rem 0 3rem 0;
+    border-radius: 10px;
+    box-shadow: 1px 1px 10px 10px rgba(0,0,0,0.1);
```
Comment (yaswanthmyneni) [inline src/Pages/SignIn/SignIn.css:24]: changes: use css variable for color here because you are using in multiple places

**Review by yaswanthmyneni — APPROVED** [review]
UI is good, you can merge after changes.

File: src/App.js [hunk 4]
```diff
@@ -1,41 +1,32 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./components/index";
+import Mockman from "mockman-js";
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { Routes , Route } from 'react-router-dom';
+import { HomePage , SignIn , SignUp , History, Likes, WatchLater, PlayList } from "./Pages/index";
```
Comment (shraddha15-creator) [inline src/App.js:7]: you don't need to mention that index at the end, by default it goes to the index file.

` import { HomePage , SignIn , SignUp , History, Likes, WatchLater, PlayList } from "./Pages"; `
would work

File: src/Pages/HomePage/HomePage.css [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.home-section{
+    width: 100vw;
+}
+
+.videos-container{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
```
Comment (shraddha15-creator) [inline src/Pages/HomePage/HomePage.css:22]: ` flex-direction: row ` is the default property so don't need to mention

**Review by shraddha15-creator — APPROVED** [review]
code is clean
good folder structure

File: src/App.js (hunk 4 above)
Comment (omkarpatke) [reply src/App.js:7]: thanks for the suggestion.

File: src/App.js [hunk 6]
```diff
@@ -1,41 +1,32 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./components/index";
+import Mockman from "mockman-js";
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```
Comment (gouravmarch20) [inline src/App.js:5]: define it in components / utils folder --> and use it as component

File: src/components/categoryBar/category.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useVideos } from '../../context/videosContext';
+import './category.css';
+
+export function CategoryBar() {
+  const { setToggleTab, toggleTab }  = useVideos();
```
Comment (gouravmarch20) [inline src/components/categoryBar/category.js:6]: naming may improve --> useState , useContext difference easy

**Review by gouravmarch20 — COMMENTED** [review]
ready to deply

**Pull request merged**